docker/Dockerfile

```diff
@@ -55,11 +25,6 @@ RUN mkdir -p /opensearch-benchmark/.benchmark && \
 
 USER 1000
 
-ENV PATH=/opensearch-benchmark/venv/bin:$PATH
-
-ARG VERSION
-ARG BUILD_DATE
-
```

Comment:
We recently improved our build repo docker file by adding the dockerfile link to label, can you do the same as here: https://github.com/opensearch-project/opensearch-build/blob/main/docker/release/dockerfiles/opensearch.al2.dockerfile#L109?

Thanks.